manala.skeleton/defaults/main.yml

```diff
@@ -61,6 +61,7 @@ manala_skeleton_options:
   influxdb:              false
   dnsmasq:               false
   sqlite:                false
+  httpie:                false
```

Comment:
This is unused, and i don't think it must be used :)

I was more thinking about something like "thefuck", an always available package